code/__DEFINES/combat.dm

```diff
@@ -63,13 +63,21 @@
 #define THROWN_PROJECTILE_ATTACK 4
 #define LEAP_ATTACK 5
 
+//Embedded objects
+#define EMBEDDED_PAIN_CHANCE 					15	//Chance for embedded objects to cause pain (damage user)
+#define EMBEDDED_ITEM_FALLOUT 					5	//Chance for embedded object to fall out (causing pain but removing the object)
+#define EMBED_CHANCE							45	//Chance for an object to embed into somebody when thrown (if it's sharp)
+#define EMBEDDED_PAIN_MULTIPLIER				2	//Coefficient of multiplication for the damage the item does while embedded (this*item.w_class)
+#define EMBEDDED_FALL_PAIN_MULTIPLIER			5	//Coefficient of multiplication for the damage the item does when it falls out (this*item.w_class)
+#define EMBEDDED_IMPACT_PAIN_MULTIPLIER			4	//Coefficient of multiplication for the damage the item does when it first embeds (this*item.w_class)
+#define EMBED_THROWSPEED_THRESHOLD				4	//The minimum value of an item's throw_speed for it to embed (Unless it has embedded_ignore_throwspeed_threshold set to 1)
+#define EMBEDDED_UNSAFE_REMOVAL_PAIN_MULTIPLIER 8	//Coefficient of multiplication for the damage the item does when removed without a surgery (this*item.w_class)
+#define EMBEDDED_UNSAFE_REMOVAL_TIME			30	//A Time in ticks, total removal time = (this*item.w_class)
+
 //Gun Stuff
  #define SAWN_INTACT  0
  #define SAWN_OFF     1
 
  #define WEAPON_LIGHT 0
  #define WEAPON_MEDIUM 1
- #define WEAPON_HEAVY 2
-
-// Embedded objects
-#define EMBED_THROWSPEED_THRESHOLD 15
```

Comment:
At first I thought something went horrendously wrong with the refactor; then I noticed this.